webapp/src/css/material.less

```diff
@@ -31,8 +31,8 @@
     div {
       flex-grow: 100;
     }
-    &.age {
-      span.relative-date.future {
+    span.relative-date.future {
+      span[data-date-options*="\"class\":\"age\"}"]{
```

Comment:
This is quite complicated and hard to read - it took me a while to figure out where the "data-date-options" was coming from. How about modifying the `getRelativeDate` function in `webapp/src/js/filters/date.js` so if `class` is passed as an option it's appended to the `classes` array and gets added to the `span`?

Then this rule would be:

```css
.relative-date.future.age {
  color: @error-color;
}
```

What do you think?

Reply:
Sure,I think that would be better.Let me implement that now